### PR TITLE
Adding a few craft that have flown over

### PR DESCRIPTION
**Pull request by challgren**
## Describe your changes

PLANE INFO: Adding some craft that have flown overhead

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

File: plane-alert-db.csv [hunk 1]
```diff
@@ -13116,6 +13122,7 @@ A4F203,N418BT,Bridger Air Tanker 3 LLC,Canadair CL-215T,CL2T,Civ,Flying Fire Ext
 A50338,N422BT,Bridger Air Tanker 6 LLC,Canadair CL-215T,CL2T,Civ,Flying Fire Extinguisher,Super Scooper,Aerial Firefighter,Sam Tân,https://en.wikipedia.org/wiki/Canadair_CL-215
 A7C4E6,N60A,5-State Helicopters Inc,Sikorsky UH-60A Black Hawk,UH60,Civ,Flying Fire Extinguisher,Aerial Firefighter,Wildfire,Sam Tân,https://en.wikipedia.org/wiki/Sikorsky_UH-60_Black_Hawk
 A1DCB0,N219TX,Texas Department of Public Safety,Pilatus PC-12,PC12,Pol,Police Squad,Spectre 219,Speed Enforced by Aircraft,Police Forces,https://en.wikipedia.org/wiki/Pilatus_PC-12
+A0644D,N124TX,Texas Department of Public Safety,Aerospatiale AS-350 Ecureuil,AS50,Pol,Police Squad,Spectre 219,Speed Enforced by Aircraft,Police Forces,https://en.wikipedia.org/wiki/Eurocopter_AS350_%C3%89cureuil
```
Comment (Phaeton) [inline plane-alert-db.csv:13125]:
```suggestion
A0644D,N124TX,Texas Department of Public Safety,Eurocopter AS350 Squirrel B2,AS50,Pol,Police Squad,Spectre 219,Speed Enforced by Aircraft,Police Forces,https://w.wiki/7kcW

```

File: plane-alert-db.csv [hunk 2]
```diff
@@ -13324,4 +13331,6 @@ ADFE4A,94-00318,United States Army,MC-12S Liberty,BE20,Mil,Surveillance,Eye In T
 AE093A,00-01051,United States Army,Cessna UC-35B Citation,C560,Mil,VIP Transport,Bizjet,Must Be Nice,Toy Soldiers,https://www.army.mil
 AE6D7E,19-5948,USAF,HC-130J Hercules,C30J,Mil,Combat King II,Search And Rescue,Air2Air,USAF,https://en.wikipedia.org/wiki/Lockheed_HC-130
 AE4FB1,168097,United States Navy,Sikorsky MH-60R Seahawk,H60,Mil,ASW,Multi Role,Sub Hunter,United States Navy,https://en.wikipedia.org/wiki/Sikorsky_SH-60_Seahawk#MH-60R
-AE4EEE,10-0055,United States Air Force,Bell/Boeing CV-22B Osprey,V22,Mil,352nd Special Operations Wing,Tiltrotor,Nope Mobile,Special Forces,https://w.wiki/3ksn
\ No newline at end of file
+AE4EEE,10-0055,United States Air Force,Bell/Boeing CV-22B Osprey,V22,Mil,352nd Special Operations Wing,Tiltrotor,Nope Mobile,Special Forces,https://w.wiki/3ksn
+A727BD,N560JL,Express Air Medical Transport,B200 Super King Air,BE20,Civ,Air Ambo,Medical Evac,Saving Lives,Flying Doctors,https://en.wikipedia.org/wiki/Beechcraft_Super_King_Air
```
Comment (Phaeton) [inline plane-alert-db.csv:13335]: This one is confusing. A727BD, N560JL is a Cessna Citation V, C560. Looks to be registered to Care Flight LLC.

**Review by Phaeton — CHANGES_REQUESTED** [review]
Got any photos for these? :innocent:

**Comment by challgren** [comment]
> Got any photos for these? 😇

Let me see what I can come up with. Wanted to clean up my list.

File: plane-alert-db.csv (hunk 1 above)
Comment (pmd5700) [reply plane-alert-db.csv:13125]: https://www.helis.com/spotters/zane_adams/as350b2_n124tx.jpg

File: plane-alert-db.csv (hunk 2 above)
Comment (pmd5700) [reply plane-alert-db.csv:13335]: AE4EEE,https://cdn.jetphotos.com/full/5/12061_1441771997.jpg,,,

Comment (dziban303) [reply plane-alert-db.csv:13335]: > AE4EEE,https://cdn.jetphotos.com/full/5/12061_1441771997.jpg,,,

already in the database

**Pull request merged**